test/regressions/tests/TextField/TextFieldMargin.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+import TextField from 'material-ui/TextField';
+
+export default function TextFieldMargin() {
```

Comment:
I don't think that we need that regression test as we are also taking into account the demo of the documentation.